server/src/creators.js

```diff
@@ -9,13 +9,15 @@ const creators = {
   organisation: ({ name }) =>
     Organisation.create({ name }),
 
-  schedule: ({ name, details, group }) => {
+  schedule: ({ name, details, startTime, endTime, group }) => {
```

Comment:
enforce the existence of startTime/endTime because (at least for now) a time segment shouldn't exist without them

Reply:
done.